contracts/strategies/morpho/MetaMorphoStrategy.sol

```diff
@@ -283,15 +283,16 @@ contract MetaMorphoStrategy is ERC4626Strategy {
         _aprData.add = _calcMarketAdd(_mmVault, _marketId, _aprData.supplyLen, _aprData.add);
       }
       // calc how much of `sub` will be removed from this market
+      uint256 _toSub;
       if (_aprData.sub > 0) {
-        _aprData.sub = _calcMarketSub(_mmVault, _marketId, _aprData.withdrawLen, _aprData.sub);
+        (_toSub, _aprData.sub) -= _calcMarketSub(_mmVault, _marketId, _aprData.sub);
```

Comment:
Why the `-=`? Shouldn't this be simply `=` ?

Reply:
Yes it should only be `=`, my bad